package.json

```diff
@@ -13,9 +13,12 @@
   },
   "devDependencies": {
     "browserify": "latest",
+    "chai": "^3.5.0",
     "derequire": "latest",
-    "watchify": "latest",
-    "uglifyjs": "latest"
+    "jsdom": "^9.12.0",
+    "mocha": "^3.2.0",
+    "uglifyjs": "latest",
```

Comment:
The deps were out of order and npm reordered them. I didn't touch uglifyjs or watchify.